core/app.js

```diff
@@ -157,6 +157,22 @@ app.use(_.get('/edit-catalog', function *() {
   });
 }));
 
+app.use(_.get('/inventory.csv', function *() {
```

Comment:
You'll want to make sure this is limited to only administrators. Otherwise, if the data is meant to be public, we'll want to make sure we explicitly name the columns we select to make sure that any (future) sensitive data isn't leaked.